libs/barista-components/quick-filter/src/quick-filter.spec.ts

```diff
@@ -35,6 +46,26 @@ import { of } from 'rxjs';
 import { DtQuickFilter, DtQuickFilterChangeEvent } from './quick-filter';
 import { DtQuickFilterDefaultDataSource } from './quick-filter-default-data-source';
 import { DtQuickFilterModule } from './quick-filter.module';
+import { DtDrawer } from '@dynatrace/barista-components/drawer';
+
+export function createFixture<T>(
+  component: Type<T>,
+  selector?: string,
+): {
+  fixture: ComponentFixture<T>;
+  instance: T;
+  containerEl: HTMLElement;
+} {
+  const fixture = TestBed.createComponent(component);
+  const container = selector
+    ? fixture.debugElement.query(By.css(selector)).nativeElement
+    : undefined;
+  return {
+    fixture,
+    instance: fixture.debugElement.componentInstance,
+    containerEl: container,
+  };
+}
```

Comment:
There is already a createComponent from `'@dynatrace/testing/browser'`. Please consider using this one instead of a custom fixture creation,

Reply:
thanks, I will adapt the implementation to those APIs.